frontend/server/src/DAO/Problems.php

```diff
@@ -400,6 +405,7 @@ function ($clause) {
         }
         $sql .= ';';
 
+        /** @var array<int, array{name: string, autogenerated: int}> $rs */
```

Comment:
```suggestion
        /** @var array{name: string, autogenerated: int}[] */
```
en general si tienes un arreglo normalito hay que usar `T[]`. `array<int, T>` hay que usarlo sólo cuando es conceptualmente un diccionario que relaciona enteros no-contiguos (ids, puntajes, o cualquier cosa que no sean índices normalitos) con `T`s.

igual en los demás lugares donde de utiliza `array<int, T>`.

también, si es una asignación, no es necesario ponerle el nombre de la variable a menos que Psalm se queje.